src/executor/logical_tile.cpp

```diff
@@ -437,7 +439,36 @@ std::vector<std::vector<std::string>> LogicalTile::GetAllValuesAsStrings() {
 	      if (base_tuple_id == NULL_OID) {
 	        row.push_back(ValueFactory::GetNullValueByType(
 	                  cp.base_tile->GetSchema()->GetType(cp.origin_column_id)).ToString());
-	      } else {
+	      } else if(cp.base_tile->GetSchema()->GetType(cp.origin_column_id) == VALUE_TYPE_TIMESTAMP){
+          auto time_str = cp.base_tile->GetValue(base_tuple_id, cp.origin_column_id).ToString();
+          
+          long long epoch = std::stoll(time_str);
```

Comment:
Wrong place to add this logic. Make this an utility function in `executor_utils`.

Add vertical spacing and comments !!!!